CONTRIBUTING.md

```diff
@@ -1,3 +1,113 @@
 # Contributing guide
 
-Page in construction, for now go to https://github.com/pymc-devs/pymc-experimental#questions.
+Thank you for your interest in contributing to PyMC and PyMC-experimental!
+
+This page outlines the steps to follow if you wish to contribute to the pymc-experimental repo and clone the repo locally.
+
+## Install locally
+**1**: Create a folder `pymc-devs` in your local machine and follow the steps on [cloning PyMC locally](https://www.pymc.io/projects/docs/en/latest/contributing/pr_tutorial.html).
+
+Since PyMC-experimental should integrate with the latest version of PyMC, it is recommended that any development work on PyMC-experimental must also work with the latest version of PyMC.
```

Comment:
I don't think that's true, don't we pin a version?

Reply:
We don't but we test against the latest